WDE.DatabaseEditors.WPF/Views/TemplateDbTableEditorView.xaml

```diff
@@ -0,0 +1,125 @@
+﻿<UserControl x:Class="WDE.DatabaseEditors.WPF.Views.TemplateDbTableEditorView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:local="clr-namespace:WDE.DatabaseEditors.WPF.Views"
+             xmlns:viewHelpers="clr-namespace:WDE.Common.WPF.ViewHelpers;assembly=WDE.Common.WPF"
+             xmlns:helpers="clr-namespace:WDE.DatabaseEditors.WPF.Helpers"
+             xmlns:dataModels="clr-namespace:WDE.DatabaseEditors.Models;assembly=WDE.DatabaseEditors"
+             mc:Ignorable="d"
+             d:DesignHeight="300" d:DesignWidth="300"
+             x:Name="rootWindow">
+    <UserControl.InputBindings>
+        <KeyBinding Modifiers="Control" Key="Z" Command="{Binding Undo}"/>
+        <KeyBinding Modifiers="Control" Key="Y" Command="{Binding Redo}"/>
+    </UserControl.InputBindings>
+    <UserControl.Resources>
+        <viewHelpers:InvertedBoolConverter x:Key="InvertedBoolConverter" />
+    </UserControl.Resources>
+    <DockPanel LastChildFill="True" HorizontalAlignment="Stretch">
+        <Grid DockPanel.Dock="Top" HorizontalAlignment="Stretch">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="10" />
+                <RowDefinition Height="*" />
+            </Grid.RowDefinitions>
+            <StackPanel Grid.Row="0">
+                <!-- <Button Content="Add new" Width="30"/> -->
+            </StackPanel>
+            <viewHelpers:StretchedTreeView Grid.Row="1" ItemsSource="{Binding TableData.Categories}" VerticalContentAlignment="Stretch" HorizontalContentAlignment="Stretch">
```

Comment:
We definitely have to change the view not to use TreeView. There are few visual bugs because of this, it is hard to extend, this is hard to move to Avalonia, this is harder to style.